support/docker-bake.hcl

```
@@ -18,6 +18,14 @@ variable "PRERELEASE" {
   default = ""
 }
 
+variable "ALL_SUPPORTED_DISTROS" {
+  default = ["alpine", "buster", "bullseye", "bookworm"]
```

Would this also work, so we don't repeat ourselves?
```suggestion
  default = concat(["alpine"], DEBIAN_DISTROS)
```

---

As discussed only using bookworm, so this won't be necessary.